app/models/loan.rb

```diff
@@ -66,8 +67,12 @@ class Loan < ActiveRecord::Base
   # beware, the methods generated by this include will fail without the corresponding OptionSet records existing in the database
   attr_option_settable :status, :loan_type, :project_type, :public_level
 
-  # results in linked CustomModel instances with the associated CustomFieldSet schema
-  attr_custom_model_linkable :loan_criteria, :loan_post_analysis
+  # provides acess to linked CustomModel instances with the associated CustomFieldSet schema.
+  # note these instances will be 'owned' by the loan's organization and potentially shared by more than one loan
```

Comment:
I believe that we are still waiting for confirmation from Brendan if he wants the questionnaire data duplicated, and the initial feedback was to just ignore the cross-links.